frontend/src/components/eMIB/Email.jsx

```diff
@@ -119,7 +146,7 @@ class Email extends Component {
         </div>
         <hr style={styles.titleEmailDivider} />
         <EmailContent email={email} />
-        <div>
+        <div id="action-views" tabIndex={TABINDEX}>
```

Comment:
You should use references instead of grabbing an element by Id: https://reactjs.org/docs/refs-and-the-dom.html

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/234/commits/b80880559ee9cf23db305e14b82b683a209c9be8